components/scream/src/physics/p3/tests/p3_tests.cpp

```diff
@@ -13,7 +13,7 @@ TEST_CASE("FortranDataIterator", "p3") {
   using scream::p3::ic::Factory;
   const auto d = Factory::create(Factory::mixed);
   scream::p3::FortranDataIterator fdi(d);
-  REQUIRE(fdi.nfield() == 42);
+  REQUIRE(fdi.nfield() == 43);
```

Comment:
Geez.

Reply:
Geez, it's lame to have to specify how many fields we pass, or it's lame to have that many fields in the first place?

Reply:
I'll put it this way: I don't ever want to understand what it's like to feel okay about passing that many parameters to a function. :-)